Dockerfile

```diff
@@ -50,7 +51,7 @@ RUN \
     cp -f ./bazel-bin/src/bindings/pydp.so ./pydp && \
     rm -rf dist/ && \
     pipenv run python3 setup.py bdist_wheel && \
-    pip3 install dist/*.whl
+    pipenv install dist/*.whl
```

Comment:
If we do this. then when we open the bash, would the pipenv env open or the default python env?
The question is. if I type 
```
python3
>> import pydp as dp
```
Would this work?

Reply:
I believe we should install the final package into a virtual environment. If you don't want the user to have to run `pipenv shell` and to simply run `python` then I think this would work.
```
ENV PIPENV_VENV_IN_PROJECT=true 
# This `activates` the virtual env
ENV VIRTUAL_ENV=$PROJECT_DIR/.venv
ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```
However, to just install it outside a virtual env is not ideal and could have unforeseen issues. We could always add a line to the .bashrc file to put the user into the relevant environment?
Perhaps I need to full understand the use of these images and I can do my best to make them work as expected.

My current understanding of what we want out of this is: 
-  Publish docker images to Docker Hub so it's easy for people to try out the library.